### DISCUSSION TITLE
Additional Relay Servers

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk-server-pro#78] [Q&A]
I will try to add more relay servers in the coming days.

I wonder why it was chosen to rely on geolocation and not latency when choosing the best relay server.

Geographical proximity is not an indication of bandwidth speed.

**Comment by Sebshino** [comment]
Indeed, I agree with you, distance does not systematically mean higher latency.  This would perhaps allow better management of flows
